src/main/java/com/programmers/heycake/domain/BaseEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.programmers.heycake.domain;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import lombok.Getter;
+
+@Getter
+@EntityListeners(AuditingEntityListener.class)
+@MappedSuperclass
+public abstract class BaseEntity {
+
+	@Column(name = "created_at", nullable = false)
```

Comment:
```suggestion
@Column(updatable = false...)

```
추가해주면 좋을 거 같습니다.
[참고](https://wakestand.tistory.com/935)

Reply:
반영했씁니다~